nms/app/packages/magmalte/alerts/xwfmAlerts.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+import type {prom_alert_config} from '@fbcnms/magma-api';
+
+export const xwfmAlerts: {[string]: prom_alert_config} = {
```

Comment:
minor nit:instead of hardcoding this, should we instead have it as some json file in NMS directory which we can import and use ? That might be more flexible

Reply:
Maybe. That seems like more of a significant change so let's just see if that's actually requested and we can implement it then.